tests/fixtures/test_app_spec.json

```diff
@@ -33,6 +33,15 @@
                 { "clientId": "client_string3",   "data": "This is a string clientData payload #3" },
                 { "clientId": "client_json",      "data": "{ \"test\": \"This is a JSONObject clientData payload\"}" }
             ]
+        },
+        {
+            "name": "persisted:presence_fixtures_ecrypted",
+            "presence": [
+                { "clientId": "client_enc1", "data": "HO4cYSP8LybPYBPZPHQOtmHItcxYdSvcNUC6kXVpMn0VFL+9z2/5tJ6WFbR0SBT1xhFRuJ+MeBGTU3yOY9P5ow==", "encoding": "utf-8/cipher+aes-128-cbc/base64" },
+                { "clientId": "client_enc2", "data": "HO4cYSP8LybPYBPZPHQOtuB3dfKG08yw7J4qx3kkjxdW0eoZv+nGAp76OKqYQ327", "encoding": "cipher+aes-128-cbc/base64" },
+                { "clientId": "client_enc3", "data": "HO4cYSP8LybPYBPZPHQOtuD53yrD3YV3NBoTEYBh4U0N1QXHbtkfsDfTspKeLQFt", "encoding": "json/utf-8/cipher+aes-128-cbc/base64" },
+                { "clientId": "client_enc4", "data": "HO4cYSP8LybPYBPZPHQOtvmStzmExkdjvrn51J6cmaTZrGl+EsJ61sgxmZ6j6jcA", "encoding": "json/utf-8/cipher+aes-128-cbc/base64" }
```

Comment:
@mattheworiordan Sorry, that's because of the PR, which changed directory structure.
[CapabilityTest](https://github.com/ably/ably-php/blob/master/tests/CapabilityTest.php)
[PHP fixtures](https://github.com/ably/ably-php/blob/master/tests/fixtures/test_app_spec.json)
I didn't write the test, but it seems to copy the test from Java lib, which is also using [these fixtures](https://github.com/ably/ably-java/blob/master/test/io/ably/test/assets/testAppSpec.json). Yep, I'll do a PR.